systems/robotInterfaces/@Biped/planFootsteps.m

```diff
@@ -91,6 +91,15 @@
 
 weights = getFootstepOptimizationWeights(obj);
 
+% Shift the problem to always start from x,y,z,yaw = 0;
+T = makehgtform('zrotate', -start_pos.right(6), 'translate', -start_pos.right(1:3));
```

Comment:
never saw that one before.  everywhere else we've been using rotZ.m and translate.m from our util directory.